### DISCUSSION TITLE
Having problems resetting the st2admin password

### DISCUSSION THREAD

**Discussion by planet** [StackStorm/st2#5965] [Q&A]
Following these directions - https://docs.stackstorm.com/authentication.html , I reset the account password for st2admin but after if it restart the st2api/st2auth service all st2ctl commands will now fail until I reset it back to the old password.  Am I missing something?

I get this error 

```s/jira_report.py -t pe -s"
Failed to authenticate with credentials provided in the config.
ERROR: 401 Client Error: Unauthorized
MESSAGE: Invalid or missing credentials for url: http://127.0.0.1:9100/tokens

```

**Comment by nzlosh** [comment]
Try checking the log for details related to the error.

**Reply by planet** [reply]
Which log?  I tried to restart the st2 services and everything.  What is the actual process to change the password using the htpassword backend?

**Comment by nzlosh** [comment]
The logs will be in journald or under `/var/log/st2` depending on your environment.  You can read about how authentication works here https://docs.stackstorm.com/authentication.html#authentication

**Comment by planet** [comment]
I figured it out, I had to relog in with st2 login.
